src/poetry_plugin_export/exporter.py

```diff
@@ -161,11 +154,9 @@ def _export_requirements_txt(self, cwd: Path, output: IO | str) -> None:
 
                     hashes.append(f"{algorithm}:{h}")
 
-                if hashes:
-                    line += " \\\n"
-                    for i, h in enumerate(hashes):
-                        suffix = " \\\n" if i < len(hashes) - 1 else ""
-                        line += f"    --hash={h}{suffix}"
+                for h in hashes:
+                    line += f" \\\n    --hash={h}"
```

Comment:
Why was this changed? This was an upstream fix in the in-tree export command iirc..

Reply:
Isn't the that equivalent?

Before change:
If there are hashes a newline is appended first. Then, each hash is appended and after each hash a newline. Except for the last hash: For the last hash no new line is appended.

After the change:
For each hash a newline is appended first and afterwards the hash.

Maybe I'm blind right now, but I don't see any difference.

Reply:
You are correct, my bad. I was tired when I looked at the code last, and it reminded me of another merge conflict I had looked at before.